cmd/agent/app/settings/runtime_settings_test.go

```diff
@@ -1,113 +1,17 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Why remove the copyright header? This applies to a few other files, I think we want this everywhere.

Reply:
This file was moved and then part of it moved back, which is how it lost the header. I'll restore it.